pkg/log/multislogger/syslogger_windows.go

```diff
@@ -0,0 +1,45 @@
+//go:build windows
+// +build windows
+
+package multislogger
+
+import (
+	"context"
+	"io"
+	"log/slog"
+
+	"github.com/kolide/launcher/pkg/log/eventlog"
+	"golang.org/x/sys/windows"
+)
+
+const serviceName = "launcher"
+
+func SystemSlogger() (*MultiSlogger, io.Closer, error) {
+	if !windows.GetCurrentProcessToken().IsElevated() {
+		syslogger := defaultSystemSlogger()
```

Comment:
This is an interesting idea. Is the intent to try to capture people running in a terminal? Im game for trying this. I don't know if we have better rope to know what our caller is